src/o365/outlook/commands.ts

```diff
@@ -2,6 +2,7 @@ const prefix: string = 'outlook';
 
 export default {
   OUTLOOK_MAIL_SEND: `${prefix} mail send`,
+  OUTLOOK_REPORT_MAILBAPPUSAGEUSERDETAIL: `${prefix} report mailappusageuserdetail`,
```

Comment:
Typo in `OUTLOOK_REPORT_MAILBAPPUSAGEUSERDETAIL`. Fixed to `OUTLOOK_REPORT_MAILAPPUSAGEUSERDETAIL`